packaging/datadog-agent/source/supervisord.conf

```diff
@@ -45,7 +45,7 @@ command=python agent/jmxfetch.py
 stdout_logfile=supervisord/logs/jmxfetch.log
 redirect_stderr=true
 priority=999
-startsecs=0
+startsecs=3
```

Comment:
By default, `startretries` is set to 3 .
The reason why I switched `startsecs` to 3, is because the Supervisor ignores `startretries` parameter when the program is able to start (i.e runs more than `startsecs` seconds).